src/main/java/duke/ui/Ui.java

```diff
@@ -0,0 +1,129 @@
+package duke.ui;
+
+import java.util.ArrayList;
+
+import duke.exception.DukeException;
+import duke.task.Task;
+import duke.task.TaskList;
+
+/**
+ * Ui create messages that are being display to user.
+ */
+public class Ui {
+
+    /**
+     * Returns exit message when user exit the programs.
+     *
+     * @return Exit message when user exit the programs.
+     */
+    public String showExit() {
```

Comment:
Maybe `exitMessage` would be much more specific to the intentions of this method?